website/snippets/available-git-providers.md

```diff
@@ -0,0 +1 @@
+When you develop in dbt Cloud, you can leverage [Git](/docs/collaborate/git-version-control) to version control your code.  To connect to a repository, you can either set up a dbt Cloud-hosted [managed repository](/docs/collaborate/git/managed-repository) or directly connect to a [supported git provider](/docs/collaborate/git/connect-github).
```

Comment:
Should we add a sentence describing the benefits of each? I assume that connecting directly to a supported git provider gives you more flexibility if you're used to the git workflow and the managed repo abstracts the git workflow if you are not ready to ramp up on the git flow?

Asked this question in [internal Slack](https://dbt-labs.slack.com/archives/C02HE19D51R/p1674148236019579).

Reply:
thanks @runleonarun - great shout! i've added a blurb on the benefits of each. let me know what you think!

Reply:
>To connect to a repository, you can either set up a dbt Cloud-hosted [managed repository](/docs/collaborate/git/managed-repository) or directly connect to a [supported git provider](/docs/collaborate/git/connect-github). Managed repositories are a great way to trial dbt without needing to create a new repository.  In the long run, it's better to connect to a supported git provider to use features like automation and [continuous integration](/docs/deploy/cloud-ci-job).